### ISSUE TITLE
Vertical scroller

### ISSUE THREAD

**Issue by gaurapanasenko** [t4t5/sweetalert#521]
Hi. I want share how i did vertical scrolling.

``` css
.sweet-alert {
    box-sizing : border-box;
    max-height : 100% !important;
    overflow-y : auto !important;
    padding : 0 17px 17px !important;
    width : 512px;
}
.sweet-alert:before {
    content : "";
    display : block;
    height : 17px;
    width : 0;
}
```

``` javascript
$(window).resize(function(){
    $(".sweet-alert").css("margin-top",-$(".sweet-alert").outerHeight()/2);
});
```

**Comment by unfomo1** [comment]
awesome, thanks!!

Noob q but ..  .. if I have a lot of text, how do I add the scrolltop() function here . . . what and how do I add it . .  .

**Comment by rankill** [comment]
To pin the modal in the middle, avoiding the margin positioning, this could be an additional code:
```
.sweet-alert {
    box-sizing : border-box;
    max-height : 100% !important;
    overflow-y : auto !important;
    padding : 0 17px 17px !important;
    width : 512px;

    //As default top and left are 50%, so it will transform those values and will set the modal exactly in the middle
    transform: translate(-50%, -50%);

   //remove margins
   margin: 0;
}

```

Note: this is just an opinion, and possibly a workaround to get vertical scroller easily

**Comment by TeyanWar** [comment]
Thanks a lot men

**Comment by StreetRock** [comment]
Thx, exactly what I was looking for!

**Comment by goktugerce** [comment]
@unfomo1 Were you able to solve that problem?

**Comment by t4t5** [comment]
Thanks for the temporary fix! The issue should be resolved in the new 2.0 release.
